### PR TITLE
Update return block for lambda modules

### PR DESCRIPTION
**Pull request by GomathiselviS**
##### SUMMARY

This  PR updates the return blocks for lambda modules.
Refer:  https://issues.redhat.com/browse/ACA-1478

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by abikouo — APPROVED** [review]

File: plugins/modules/lambda_event.py [hunk 1]
```diff
@@ -144,10 +144,109 @@
...
+            type: dict
+            contains:
+              destination:
+                description: Destination resource ARN.
+                type: str
+      event_source_arn:
+        description: The Amazon Resource Name (ARN) of the event source.
+        type: str
+        returned: always
+        sample: "arn:aws:dynamodb:us-west-2:xxxxxx:table/yyyy/stream/2024-06-06T07:03:58.956"
```
Comment (gravesm) [inline plugins/modules/lambda_event.py:184]: I'm not sure if we have a practice around example ARNs, but I'd suggest using `123456789012` for the account number.

File: plugins/modules/lambda_event.py [hunk 2]
```diff
@@ -144,10 +144,109 @@
...
+      function_arn:
+        description: The ARN of the Lambda function.
+        type: str
+        returned: always
+        sample: "arn:aws:lambda:us-west-2:721066863947:function:test-lambda-3a0a29d0dbb5:1"
```
Comment (gravesm) [inline plugins/modules/lambda_event.py:189]: Same here.

**Review by gravesm — COMMENTED** [review]
A few optional comments.

**Review by gravesm — APPROVED** [review]

File: plugins/modules/lambda_event.py (hunk 1 above)
Comment (tremble) [reply plugins/modules/lambda_event.py:184]: I don't think I documented it properly, but I went through and cleaned up a lot of things like that, `123456789012` is the example account I was using.  (Cleaning up the examples helped keep InfoSec off my back with false positives of "leaked" credentials/resource IDs).  https://github.com/ansible-collections/amazon.aws/pull/1070/files.

I should probably get around to fixing the lack of documentation...

**Comment by GomathiselviS** [comment]
regate

**Comment by abikouo** [comment]
regate

**Comment by GomathiselviS** [comment]
regate

**Pull request merged**